docs/static/troubleshoot/ts-logstash.asciidoc

```diff
@@ -72,6 +72,39 @@ Try adding these values to the `jvm.options` file.
 to share, please comment in the
 https://github.com/elastic/logstash/issues/10496[issue].
 
+
+[[ts-windows-permission-denied-NUL]]
+===== 'Permission denied - NUL' errors on Windows
+
+* When running Logstash using certain user supplied versions of the JDK on Windows, Logstash may not be able to start, instead
+  failing with the following error:
+
+```
+[FATAL] 2022-04-27 15:13:16.650 [main] Logstash - Logstash stopped processing because of an error: (EACCES) Permission denied - NUL
+org.jruby.exceptions.SystemCallError: (EACCES) Permission denied - NUL
+```
```

Comment:
```suggestion
*Sample error*

[source,sh]
-----
[FATAL] 2022-04-27 15:13:16.650 [main] Logstash - Logstash stopped processing because of an error: (EACCES) Permission denied - NUL
org.jruby.exceptions.SystemCallError: (EACCES) Permission denied - NUL
-----

```